sites/friday/src/get-started/express/_index.md

```diff
@@ -1,4 +1,490 @@
+{{< /note >}}
+
+{{% guides/requirements name="Express" %}}
+
+## Create your local Express app
+First thing’s first, if you don’t have a local Express project, you need to create a new Express project locally following their [installation guide](https://expressjs.com/en/starter/installing.html).
+
+Please refer to all the steps of the official Express installation guide for further details, but to sum it up, this is the 4 steps to create an Express app locally:
+
+```shell
+$ mkdir my-express-app
+$ cd my-express-app
+$ npm init
+$ npm install express
+```
+
+## Init your Git repo
+We need to initialize the local Git repository and commit local files, using the following command:
+```
+$ git init
+$ git add package.json package-lock.json
+$ git commit -m "Init Express application"
+```
+
+We should also ignore adding ``node_modules`` folder to the Git repository. Use the following commands to do so:
+
+```shell
+$ echo "/node_modules" >> .gitignore
+$ git add .gitignore && git commit -m "adding node_modules folder in .gitignore file"
+```
+
+## Add a Hello World route
+Please create your first Express page.
+To do so, please create, at the root of your project, a new ``index.js`` file that will contain a basic Hello world script:
+```javascript
+const express = require('express')
+const app = express()
+var port = (process.env.PORT || '3000');
+
+app.get('/', (req, res) => {
+  res.send('Hello World!')
+})
+
+app.listen(port, () => {
+  console.log(`Example app listening on port ${port}`)
+})
+```
+Then please commit your file using the following commands:
+```shell
+$ git add index.js
+$ git commit -m "adding index.js"
+```
+
+## Create a new project
+The next step is to create a project on {{% vendor/name %}}.
+To do this, you can either use the {{% vendor/name %}} CLI or the {{% vendor/name %}} console—details on how to do this can be found below.
+
+{{< note title="Remember" >}}
+At the end of either using the {{% vendor/name %}} CLI or the Console section, please copy your new **projectId**, we’ll use it in the next section: link your local project to your {{% vendor/name %}} project.
+{{< /note >}}
+
+{{< codetabs >}}
++++
+title=using the CLI
++++
+To create a new project with the {{% vendor/name %}} CLI, use the following command and follow the prompt:
+```shell
+$ {{% vendor/cli %}} project:create
+```
+
+{{< note >}}
+When creating a new project using the {{% vendor/name %}} CLI command ``project:create``, it will ask a question if you want to set the local remote to your new project. Please say Yes (y) to it.
+Your local source code will be automatically linked to your newly created {{% vendor/name %}} project by creating a `.{{% vendor/cli %}}/local/project.yaml` file that will contain the corresponding `<projectId>` and set a Git remote to `{{% vendor/cli %}}`.
+
```

Comment:
`upsun set-remote` will provide an interactive list to select the remote. Therefore, it is possible to set the remote without keeping track of that id. Should we skip the `**Make sure you save your projectId for later use.**` part?

Reply:
@AnouckColson 
i agree. 
In a normal way, as your local Git repo is already initialized, when doing the `project:create`, it will automatically set the remote. 
If not, they need to refer to section `Set project remote` with a tips on how to get this projectId back (`project:list`). 
So maybe it's better to remove this sentence, right?

Reply:
```
When creating a new project using the {{% vendor/name %}} CLI command `project:create`, you will be asked if you want to set the local remote to your new project. Enter **Yes (y)**.
Your local source code will be automatically linked to your newly created {{% vendor/name %}} project by creating a `.{{% vendor/cli %}}/local/project.yaml` file that will contain the corresponding `<projectId>` and set a Git remote to `{{% vendor/cli %}}`.

If you miss to say **Yes (y)**, please refer to section [Set project remote](#set-project-remote);
```